### DISCUSSION TITLE
App\Orchid\Screens\PlatformScreen not found on install

### DISCUSSION THREAD

**Discussion by vanvanni** [orchidsoftware/platform#1705] [General]
Hi,

I would like to address this issue here as well so more people can help on this. 
**Original Tweet:** https://twitter.com/xJoeyv/status/1386671809278451719

What it looks like when I try to install it doesn't publish/copy over the example files to the app directory. But the routeservice provider is registered when the optimize script is called. So it tries to find the class that isn't found.

**Comment by tabuna** [comment]
Does this happen with every installation? Even new?

**Reply by vanvanni** [reply]
Haven't test it yet. Ill try tomorrow after my exams if I can find a spot. I will keep you posted

**Reply by vanvanni** [reply]
I currently could quickly check on a clean Laravel install (not create orchid project) and that worked fine. So now I gotta know what in my project could be the issue this happens

**Reply by vanvanni** [reply]
**UPDATE:** It wasn't a bug. I had still the `platform.php` in my routes from the beginning of the project. Weird that I never had the issues before.
